client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -117,7 +118,7 @@ export const useJsonForms = (
   };
 
   const updateData = (newData: JsonData) => {
-    setIsDirty(isDirty === undefined ? false : true);
+    setIsDirty(!isEqual(data, newData));
```

Comment:
The form was insisting that I save, even after I had just saved. I found this annoying, so have popped this in..